### PR TITLE
Moved project to the spec 3 requirements

### PR DESCRIPTION
**Pull request by Buscedv**

### CODE REVIEW

**Comment by Buscedv** [comment]
The documentation i.e README will need updating.

File: personnummer/personnummer.py [hunk 1]
```diff
@@ -2,211 +2,261 @@
...
+            'year': year,
+            'month': month,
+            'day': day,
+            'sep': sep,
+            'num': num,
+            'check': check
+        }
+
+    @staticmethod
+    def luhn(data):
```
Comment (frozzare) [inline personnummer/personnummer.py]: No need for static method here right?

**Comment by frozzare** [comment]
Looks good other than the static luhn method and we should change the test cases to use [this](https://github.com/personnummer/meta/tree/master/testdata) testdata to avoid using existing numbers. The recommend way is to download the test files and use the values from there, see [php tets](https://github.com/personnummer/php/blob/master/tests/PersonnummerTest.php) for examples.

Any to add @Johannestegner?

**Comment by Buscedv** [comment]
@frozzare fixed!

**Comment by Johannestegner** [comment]
LGTM (but my python knowledge are too bad to say much, hehe).

**Comment by frozzare** [comment]
LGTM!

**Pull request merged**